docs/standard/serverless-architecture/architecture-approaches/architecture-deployment-approaches.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Serverless apps. Architecture, patterns, and Azure implementation.
+description: Serverless apps. Architecture, patterns, and Azure implementation. | Architecture deployment approaches
```

Comment:
it's in our backlog to do proper SEO for the other books but ideally it would be good to do new ones right. some guidance: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-basics?branch=master

think of it as a call to action to invite folks to come to this page